iothub/device/src/Transport/AmqpIoT/AmqpIoTTransport.cs

```diff
@@ -200,7 +200,25 @@ private bool OnRemoteCertificateValidation(object sender, X509Certificate certif
                 return true;
             }
 
+            if (!_amqpTransportSettings.CertificateRevocationCheck && sslPolicyErrors == SslPolicyErrors.RemoteCertificateChainErrors && CausedByRevocationCheckError(chain))
+            {
+                return true;
+            }
+
             return false;
         }
+
+        private bool CausedByRevocationCheckError(X509Chain chain)
```

Comment:
I think we want to ensure all status are not X509ChainStatusFlags.RevocationStatusUnknown in order to return false.  
```suggestion
        private bool CausedByRevocationCheckError(X509Chain chain) => chain.ChainStatus.Any(s => s.Status == X509ChainStatusFlags.RevocationStatusUnknown);
```

Reply:
I tested this method with my suggested change.

Reply:
We want to make sure there is no other  `X509ChainStatusFlags` but `X509ChainStatusFlags.RevocationStatusUnknown`.